### PR TITLE
Creepy changed to creeping

### LINKED ISSUE #15: Rename Creepy Anomaly to Creeping Anomaly
_Creeping Anomaly_ is better fitting name/description.

### PR DESCRIPTION
**Pull request by RedCore0**
added ing to all the creeps
hopefully haven't missed anything

---

fixes #15

### CODE REVIEW

**Comment by wenig** [comment]
@RedCore0 please fix the tests before we can merge it.

File: tests/configs/example-config-creep-variance-transition.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ timeseries:
     anomalies:
       - exact-position: 460
         length: 540
-        creep-length: 500
+        creep-lengthing: 500
```
Comment (wenig) [inline tests/configs/example-config-creep-variance-transition.yaml:11]:
```suggestion
        creeping-length: 500
```
:thinking:

**Comment by wenig** [comment]
@RedCore0 can you also change the filenames from `creep` to `creeping`?

**Comment by RedCore0** [comment]
> @RedCore0 can you also change the filenames from `creep` to `creeping`?

will do, sorry for being inactive for some time

**Comment by CodeLionX** [comment]
The pipeline is still failing. You can also run the tests locally with `python setup.py test`

**Pull request closed without merging**